js/src/setup-mc/setup-stepper/setup-free-listings/useSaveSuggestions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import { useAppDispatch } from '.~/data';
+
+/**
+ * A hook that returns a `saveSuggestions` callback.
+ *
+ * If there is an error during saving suggestions,
+ * it will display an error notice in the UI.
+ *
+ * @return {Function} `saveSuggestions` function to save suggestions as shipping rates.
+ */
+const useSaveSuggestions = () => {
```

Comment:
This is actually not a new file, the file was moved up to parent folder in [`f7ad55d` (#1218)](https://github.com/woocommerce/google-listings-and-ads/pull/1218/commits/f7ad55d2968f86b6979fe5d8164ffbed95210a0f).